src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -160,7 +170,6 @@ public function load(array $configs, ContainerBuilder $container): void
         $this->configureProviders($container, $config);
         $this->configureAdapters($container, $config);
         $this->configureResizers($container, $config);
-        $this->configureClassesToCompile();
```

Comment:
Did you remove this on the merge, or it was removed on 3.x?

Reply:
I removed it because it is dead code and not removing it would mean solving a cumbersome conflict (see above)